extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -252,6 +268,15 @@ export class AmpUserNotification extends AMP.BaseElement {
     });
   }
 
+  /**
+   * Get cid service.
+   * @return {!Promise}
+   * @private
+   */
+  getCidService_() {
+    cidForDoc(this.element);
```

Comment:
ugh, extension was not type-checked, added type checking (caught an unquoted JSON issues as well)